buildtest/cli/report.py

```diff
@@ -828,14 +852,16 @@ def report_summary(report, pager=None):
     )
     if pager:
         with console.pager():
-            print_report_summary_output(report, table, pass_results, fail_results)
+            print_report_summary_output(
+                report, table, pass_results, fail_results, color=color
+            )
 
         return
 
-    print_report_summary_output(report, table, pass_results, fail_results)
+    print_report_summary_output(report, table, pass_results, fail_results, color=color)
 
 
-def print_report_summary_output(report, table, pass_results, fail_results):
+def print_report_summary_output(report, table, pass_results, fail_results, color=None):
```

Comment:
can you add `color` in the docstring in the output below where you have 
```
    Args:
        report (buildtest.cli.report.Report): An instance of Report class
        table (rich.table.Table): An instance of Rich Table class
        pass_results (buildtest.cli.report.Report): An instance of Report class with filtered output by ``state=PASS``
        fail_results (buildtest.cli.report.Report): An instance of Report class with filtered output by ``state=FAIL``
```

Reply:
wiil do